### ISSUE TITLE
Effective searching by native language name

### ISSUE THREAD

**Issue by janpipek** [janpipek/iso639-python#3]
It's typical for the native language strings to contain more values (like my own native language, Czech). I'd like to be able to use these values for lookup.

Currently, I need to call exactly this:

```
iso639.to_name("čeština; český jazyk")
```

However, this does not work:

```
iso639.to_name("čeština")
```

@hosford42 - any idea? substrings, splitting, regex-searching, ...?

**Comment by hosford42** [comment]
The regex is sounding like the best option to maintain the current interface. We could compile it in advance to replace the condition `item[key] == val` on line 41. Something along these lines:

```
...
val = unicode(val)
pattern = re.compile('(^|;)\s*' + re.escape(val) + '\s*($|;)')
return next((item for item in data if any(pattern.match(item[key]) for key in keys)), None)
```

(I haven't properly tested this code, though.)

**Comment by hosford42** [comment]
Something else to consider: Do you want to address case sensitivity? If so, it would be a good time to do so, since the same part of the code would be affected.

**Comment by janpipek** [comment]
I agree. While it makes sense that some of the native names start with a capital letter and some don't, it should not be a query requirement. The regex seems perfect, I'll include the code and test it in a couple of hours. Thanks.

**Comment by janpipek** [comment]
Unfortunately, this works (after proper regex-escaping) only for the first item of the list, not for the others. I am looking for a solution.

**Comment by janpipek** [comment]
Solved without regex: 03547d2004e617db23182a9b0988c454a812f2fc .
